docs/source/tutorials/projects.rst

```diff
@@ -446,11 +446,11 @@ A project's ``<label_hash>`` is found in the project information ``project``, wh
     }
 
 
-Use the :class:`encord.project.Project` to fetch individual label objects.
+Use the :class:`encord.client.EncordClientProject` to fetch individual label objects.
```

Comment:
This is the project client. We should be referring to the user client wherever possible with the latest SDK. Does this code work with the user client?

Reply:
The 'correct' way to do this is use the user client and do client.get_project(<project_hash>).get_label_row(<label_hash>)

Reply:
Where client is the user_client

Reply:
Understood, my confusion arose from the ambiguous use of the term "client" in the docs, i.e from code snippets [here](https://python.docs.encord.com/authentication.html#resource-authentication)
```python
client = EncordClient.initialise(Project_ID, API_key)
project = client.get_project()  # This project object does NOT define get_label_row()
```
but then, with the new user client:
```python
user_client = EncordUserClient.create_with_ssh_private_key("<your_private_key>")
project: Project = user_client.get_project("<project_id>")  # This project object does define get_label_row()
```
Thus, when docs for `get_label_row()` say that:
![image](https://user-images.githubusercontent.com/92868166/180196896-7986cfdd-d293-4027-a581-eac2963b9a27.png)
there is bound to be confusion.

Reply:
Regardless, this PR is not required anymore.